web/pages/guests/index.tsx

```diff
@@ -70,6 +98,18 @@ export default function Page() {
             </div>
             <Link href="/guests/add"><a className="btn btn-primary mt-5">Add Family</a></Link>
             <Link href="/guests/invitations"><a className="btn btn-primary mt-5">All Invitations</a></Link>
+            <CSVDownloadButton 
+              items={guestsFlattened}
+              getRow={(item) => {
+                return {
+                  family_name: item.family.family_name,
+                  guest_first_name: item.guest.first_name,
+                  guest_last_name: item.guest.last_name,
+                  is_coming: guestResponseMap[item.guestId]?.is_coming ? 'YES' : 'NO',
+                }
+              }}
+              filename='Guests' 
+            />
```

Comment:
@brijdaniel this is where the button is used, see how you can alter the `getRow` function to change how the data is exported to CSV for each row